src/app/pages/system/general-settings/support/license/license.component.ts

```diff
@@ -49,18 +49,20 @@ export class LicenseComponent {
       this.isFormLoading = false;
       // To make sure EULA opens on reload; removed from local storage (in topbar) on acceptance of EULA
       window.localStorage.setItem('upgrading_status', 'upgrading');
-      this.dialogService.confirm({
-        title: helptext.update_license.reload_dialog_title,
-        message: helptext.update_license.reload_dialog_message,
-        hideCheckBox: true,
-        buttonMsg: helptext.update_license.reload_dialog_action,
-        hideCancel: true,
-        disableClose: true,
-      }).pipe(filter(Boolean), untilDestroyed(this)).subscribe(() => {
-        document.location.reload();
-      });
       this.slideInService.close();
       this.cdr.markForCheck();
+      setTimeout(() => {
```

Comment:
Is there another possible approach to solve this without setTimeout?

Reply:
For example, wrap `document.location.reload();` into `setTimeout()` or even add pipe `delay(200)`?

Reply:
It does not work for me. While working the `slideInService` and `dialogService` together, the subscription was destroyed.
I think should run sequentially.
How do you think to close slide after confirmation?